test/pathBuilder.test.js

```diff
@@ -71,15 +71,15 @@ describe('documentation paths', () => {
     });
 
     it('css distribution directory path should be correct', () => {
-        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/css');
+        expect(pathBuilder.docsCssDistDir).toBe('./docs/dist/assets/dist/css');
```

Comment:
This seems a bit strange to have `dist` twice in the same path.  Surely if the assets are in a `dist` folder, they don’t need a second `dist` directory within the assets folder too?

Is there a use-case in which it’s useful to have this – not sure if I’m missing something 🎱